src/lib/gundb/UserStorageClass.js

```diff
@@ -598,9 +599,20 @@ export class UserStorage {
 
       if (get(feedEvent, 'data.receipt')) {
         logger.debug('handleReceiptUpdated skipping event with receipt', feedEvent, receipt)
+
         return feedEvent
       }
 
+      const receiptLogEvents = get(receipt, 'logs[0].events') || []
```

Comment:
the data variable should have the relevant event, in this case Transfer.
we extract the events in getReceiveDataFromReceipt
so no need to get the event again. also logs[0] doesnt ensure its the transfer event.

Reply:
done